spec/features/incoming_mail_handler_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe "Handles incoming email" do
+  let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
```

Comment:
I have no preference for let-vs-private. I figure if the feature is in rspec (let) then it won't be a surprise for newcomers to see it. Private methods, while possibly more performant, feel, I dunno, less idiomatic? +0 from me at a philosophical level. Happy to follow whatever consensus emerges.